libandroid-navigation/src/test/java/com/mapbox/services/android/navigation/v5/navigation/NavigationRouteProviderTest.java

```diff
@@ -0,0 +1,52 @@
+package com.mapbox.services.android.navigation.v5.navigation;
+
+import android.location.Location;
+
+import com.mapbox.services.android.navigation.v5.route.RouteFetcher;
+import com.mapbox.services.android.navigation.v5.routeprogress.RouteProgress;
+
+import org.junit.Test;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+
+public class NavigationRouteProviderTest {
```

Comment:
👀 `NavigationRouteProvider` comments above. The tests included here maybe should be part of `RouteFetcher`.